### ISSUE TITLE
Error initializing native libSDL2-2.0.so.0 

### ISSUE THREAD

**Issue by MitchHS** [CM2Walki/steamcmd#37]
Hi, 

So I'm trying to get steamcmd running on pi4b to setup a valheim dedicated server. 

I'm following this guide: https://singleboardbytes.com/2058/set-up-a-valheim-dedicated-server-on-the-raspberry-pi.htm

Although I've ran into this issue when running steamcmd. 

``` pi@raspberrypi4b:~/SteamCMD $ chmod +x steamcmd.sh
./steamcmd.sh
Box86 with Dynarec v0.2.2 bedacef6 built on May 14 2022 12:53:15
Redirecting stderr to '/home/pi/Steam/logs/stderr.txt'
ILocalize::AddFile() failed to load file "public/steambootstrapper_english.txt".
[  0%] Checking for available update...
[----] Downloading update (0 of 54,937 KB)...
[  0%] Downloading update (1,556 of 54,937 KB)...
[  2%] Downloading update (3,628 of 54,937 KB)...
[  6%] Downloading update (5,639 of 54,937 KB)...
[ 10%] Downloading update (7,535 of 54,937 KB)...
[ 13%] Downloading update (9,605 of 54,937 KB)...
[ 17%] Downloading update (11,553 of 54,937 KB)...
[ 21%] Downloading update (13,423 of 54,937 KB)...
[ 24%] Downloading update (15,336 of 54,937 KB)...
[ 27%] Downloading update (17,365 of 54,937 KB)...
[ 31%] Downloading update (19,266 of 54,937 KB)...
[ 35%] Downloading update (21,110 of 54,937 KB)...
[ 38%] Downloading update (22,989 of 54,937 KB)...
[ 41%] Downloading update (24,894 of 54,937 KB)...
[ 45%] Downloading update (26,805 of 54,937 KB)...
[ 48%] Downloading update (28,635 of 54,937 KB)...
[ 52%] Downloading update (30,476 of 54,937 KB)...
[ 55%] Downloading update (32,432 of 54,937 KB)...
[ 59%] Downloading update (34,307 of 54,937 KB)...
[ 62%] Downloading update (36,791 of 54,937 KB)...
[ 66%] Downloading update (37,912 of 54,937 KB)...
[ 69%] Downloading update (39,203 of 54,937 KB)...
[ 71%] Downloading update (40,413 of 54,937 KB)...
[ 73%] Downloading update (41,658 of 54,937 KB)...
[ 75%] Downloading update (42,860 of 54,937 KB)...
[ 78%] Downloading update (44,240 of 54,937 KB)...
[ 80%] Downloading update (45,479 of 54,937 KB)...
[ 82%] Downloading update (46,715 of 54,937 KB)...
[ 85%] Downloading update (47,936 of 54,937 KB)...
[ 87%] Downloading update (49,176 of 54,937 KB)...
[ 89%] Downloading update (50,544 of 54,937 KB)...
[ 92%] Downloading update (51,775 of 54,937 KB)...
[ 94%] Downloading update (53,028 of 54,937 KB)...
[ 96%] Downloading update (54,260 of 54,937 KB)...
[ 98%] Downloading update (54,937 of 54,937 KB)...
[100%] Download Complete.
[----] Applying update...
[----] Extracting package...
[----] Extracting package...
[----] Extracting package...
[----] Extracting package...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Cleaning up...
[----] Update complete, launching...
Box86 with Dynarec v0.2.2 bedacef6 built on May 14 2022 12:53:15
Redirecting stderr to '/home/pi/Steam/logs/stderr.txt'
[  0%] Checking for available updates...
[----] Verifying installation...
[  0%] Downloading update...
[  0%] Checking for available updates...
[----] Download complete.
[----] Extracting package...
[----] Extracting package...
[----] Extracting package...
[----] Extracting package...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Installing update...
[----] Cleaning up...
[----] Update complete, launching Steamcmd...
Box86 with Dynarec v0.2.2 bedacef6 built on May 14 2022 12:53:15
Redirecting stderr to '/home/pi/Steam/logs/stderr.txt'
[  0%] Checking for available updates...
[----] Verifying installation...
Steam Console Client (c) Valve Corporation - version 1652487999
-- type 'quit' to exit --
Loading Steam API...OK

Steam>Error initializing native libSDL2-2.0.so.0 (last dlerror is libSDL2-2.0.so.0: cannot open shared object file: No such file or directory)
```

I've confirmed that libsdl2-2.0 is installed too. 

```pi@raspberrypi4b: $ sudo apt-get install libsdl2-2.0
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'libsdl2-2.0-0' for regex 'libsdl2-2.0'
libsdl2-2.0-0 is already the newest version (2.0.14+dfsg2-3).
0 upgraded, 0 newly installed, 0 to remove and 4 not upgraded.
```

TLDR: 
Steam>Error initializing native libSDL2-2.0.so.0 (last dlerror is libSDL2-2.0.so.0: cannot open shared object file: No such file or directory)
libsdl2-2.0-0 is installed.

**Comment by CM2Walki** [comment]
The Valheim image is in the progress of being upgraded to debian bullseye, sorry about the inconvenience.
